index.html

```diff
@@ -1,11 +1,32 @@
 <!doctype html>
 <html>
   <head>
-    <title>Donut Shops</title>
+    <title>DeLorean Dounts</title>
   </head>
   <body>
-    <h1>Donut Shops</h1>
+    <h1>DeLorean Donuts</h1>
    <table id="donut-shops"></table>
+   <form id="donut-form">
+     <label for="location-name">Loc</label>
```

Comment:
The `for` attrib in the label tag should match the `id` attrib in the input tag.